frontend/src/component/common/Table/cells/LinkCell/LinkCell.tsx

```diff
@@ -12,6 +12,10 @@ import {
     StyledDescription,
 } from './LinkCell.styles';
 
+//@ts-ignore
```

Comment:
Just curious - Why is this needed?

Reply:
because `remove-markdown` does not have typescript definitions. I should add one instead of ignoring, just did not want to do it for the PoC.